discovery/store.go

```diff
@@ -95,6 +95,23 @@ func (l presentationRefreshRecord) TableName() string {
 	return "discovery_presentation_refresh"
 }
 
+// presentationRefreshError is a record of a failed refresh attempt.
+type presentationRefreshError struct {
```

Comment:
having this as child resource of the Presentation speeds up fetching , if preloading (now getting activation status is always 1+1 SQL query)

Reply:
and cascading delete would cleanup any lingering errors

Reply:
Although foreign keys and cascading deletes are not required for the relationship on GORM level to work